main.go

```diff
@@ -219,6 +220,15 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "EdgeDeployment")
 		os.Exit(1)
 	}
+
+	// webhooks
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
Shouldn't it be used in the same way as `OBC_AUTO_CREATE`?

Reply:
we don't want to put everything in config. OBC is indeed configuration. user may want to use his own S3 solution. However, webhook is not optional. it is only optional for development.

Reply:
So are you trying to prevent disabling webhook in prod environment in this way?

Reply:
yes